pallets/gear/Cargo.toml

```diff
@@ -95,10 +95,10 @@ demo-calc-hash.workspace = true
 demo-calc-hash-over-blocks.workspace = true
 demo-calc-hash-in-one-block.workspace = true
 demo-compose.workspace = true
-demo-constructor.workspace = true
+demo-constructor = { workspace = true, features = [ "debug" ] }
 demo-mul-by-const.workspace = true
 demo-wait.workspace = true
-demo-waiter = { workspace = true, features = ["debug"] } # The 'debug' feature is required for some assertion in tests
+demo-waiter = { workspace = true, features = [ "debug" ] }
```

Comment:
I'd avoid adding whitespaces in arrays. References:

- https://toml.io/en/
- https://github.com/rust-lang/cargo/blob/master/Cargo.toml

Reply:
Sure, maybe we should get a TOML style guide?